XcodeServerSDK/Utils/JSON.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 public protocol JSONSerializable {
-    init?(json: NSDictionary)
+    init?(json: NSDictionary) throws
```

Comment:
Isn't this change going to make the whole project *not* compile until we change all the initializers?